sound/soc/sof/intel/hda-dsp.c

```diff
@@ -375,7 +366,7 @@ static int hda_resume(struct snd_sof_dev *sdev, bool runtime_resume)
 
 	/* check jack status based on controller status */
 	if (runtime_resume)
-		hda_codec_jack_check(sdev, status);
+		hda_codec_jack_check(sdev);
```

Comment:
the comment above should now be 'check jack status', we don't get the controller status any longer

Reply:
@plbossart fixed, thanks